aspen/request_processor/__init__.py

```diff
@@ -23,7 +23,7 @@
     # 'name':               (default,               from_unicode)
 KNOBS = \
     { 'changes_reload':     (False,                 parse.yes_no)
-    , 'charset_dynamic':    ('UTF-8',               parse.charset)
+    , 'charset_dynamic':    ('UTF-8',               parse.codec)
     , 'charset_static':     (None,                  parse.charset)
```

Comment:
Why do we use different parsers for the two charsets now? Shouldn't they both use the same one?